src/data/gitmojis.json

```
@@ -95,7 +95,7 @@
       "emoji": "🚨",
       "entity": "&#x1f6a8;",
       "code": ":rotating_light:",
-      "description": "Remove linter warnings.",
+      "description": "Fix compiler or linter warnings.",
```

```suggestion
      "description": "Fix compiler / linter warnings.",
```

In other gitmojis we use `/` instead of `or` like:
- `Improve structure / format of the code.`
- `Improve user experience / usability.`

Probably it is better to follow this convention 👍

---

Good point! I've incorporated your changes.